sapl/sessao/views.py

```diff
@@ -4339,3 +4339,61 @@ def get_initial(self):
 
     class DeleteView(MasterDetailCrud.DeleteView):
         pass
+
+
+def voto_nominal_parlamentar(request):
+    parlamentar_id = request.GET['parlamentar_id']
```

Comment:
Sugestão (retorna None se a chave-valor não existir):

```suggestion
    parlamentar_id = request.GET.get('parlamentar_id')
```

Reply:
Resolvido.